samples/sampler/stories/qa/Text.js

```diff
@@ -91,6 +146,79 @@ export const TallGlyphSupportInComponents = () => {
 					<br />
 					<Header alt="Header Compact" type="compact" title={children} subtitle={children} />
 				</Section>
+
+				<Section title="Different components with tall characters as children">
+					<Button
+						alt="Button"
+						onClick={action('onClick')}
+						backgroundOpacity={select('backgroundOpacity', prop.backgroundOpacity, Button)}
+						disabled={boolean('disabled', Button)}
+						focusEffect={select('focusEffect', prop.focusEffect, Button)}
+						icon={select('icon', prop.icons, Button)}
+						minWidth={boolean('minWidth', Button, true) ? void 0 : false}
+						selected={boolean('selected', Button)}
+						size={select('size', ['small', 'large'], Button)}
+					>
+						{select('value', prop.buttonTallText, Button, 'ฟิ้  ไั  ஒ  து')}
+					</Button>
+					<CheckboxItem
+						alt="CheckboxItem"
+						disabled={boolean('disabled', CheckboxItemConfig, false)}
+						inline={boolean('inline', CheckboxItemConfig)}
+						onToggle={action('onToggle')}
+					>
+						{select('children', prop.tallText, CheckboxItemConfig, prop.tallText[0])}
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/113989913-dcf74200-988b-11eb-9f47-1429d8306323.png)

it would be better if it could be changed all together by selecting from Text.